docs/database/ef-core-migrations.md

```diff
@@ -61,6 +61,22 @@ Start by creating some migrations to apply.
       - Creates a migration named *InitialCreate*.
       - Creates the migration in the in the *Migrations* folder in the *SupportTicketApi.Data* project.
 
+### Using Package Manager Console
```

Comment:
Wouldn't it be better to have **.NET API** and **Package Manager Console** tabs here? Or else have H3 headings for each of them.

Reply:
Wouldn't it be better to have .NET API and Package Manager Console tabs here? Or else have H3 headings for each of them.